build/common.project.props

```diff
@@ -87,7 +87,7 @@
     <Features>strict</Features>
     <!-- Same as SDK default, but without CandidateAssemblyFiles in front, which would search in Content items -->
     <AssemblySearchPaths>{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
-    <LangVersion>9</LangVersion>
+    <LangVersion>11</LangVersion>
```

Comment:
Why not int Directory.Build.props? I think using retail MSBuild would benetit simpliflyfing code.

Reply:
This file is imported by DBP already, so there's really no different. Keeping it in this file where all the current properties are is preferrable right now. 

At a later point someone can move these props if they wish, but that can be its own change.